src/pages/Home/components/CardsMenu/index.tsx

```diff
@@ -0,0 +1,142 @@
+import { useMemo } from 'react'
+import {
+  Card,
+  CardContent,
+  CardMedia,
+  Heading,
+  Icon,
+  Image,
+  Link,
+  themeColor,
+} from '@amsterdam/asc-ui'
+import { ExternalLink } from '@amsterdam/asc-assets'
+import styled from 'styled-components'
+import {
+  MenuOrCardItemData as CardItemData,
+  menuOrCardItems as cardItems,
+} from '../../../../shared/utils/menuOrCardItems'
+import { getGeneratedPath } from '../../../../shared/utils/path'
+
+const CARD_WIDTH_PX = 300
+
+const CardLink = styled(Link)`
+  padding: 0;
+
+  &:hover,
+  &:focus {
+    color: ${themeColor('secondary')};
+
+    & > div {
+      box-shadow: none;
+    }
+
+    img {
+      transform: scale(2) rotate(15deg);
+    }
+  }
+
+  &:focus {
+    outline: 1px solid ${themeColor('secondary')};
+  }
+`
+
+const StyledCard = styled(Card)`
+  width: ${CARD_WIDTH_PX}px;
+  aspect-ratio: 3 / 2;
+`
+
+const StyledCardMedia = styled(CardMedia)`
+  height: 100%;
+`
+
+const StyledCardContent = styled(CardContent)`
+  min-height: 0;
+`
+
+const StyledHeading = styled(Heading)`
+  display: flex;
+  gap: 0.75rem;
+  margin: 0;
+
+  & > span {
+    margin-top: 2px;
+  }
+`
+
+const StyledImage = styled(Image)`
+  transition: transform 500ms ease-out;
+`
+
+function CardsMenu() {
+  const cardItemsWithPaths = useMemo<CardItemData[]>(() => {
+    return cardItems.map(card => {
+      const cardWithPath: CardItemData = {
+        title: card.title,
+        description:
+          card.description ??
+          'In irure do consequat eiusmod eiusmod incididunt velit quis sint officia enim duis. Velit sunt veniam cillum culpa deserunt sit occaecat cillum enim consequat ea sit sunt.',
+        path: card.path ?? '',
+      }
+
+      if (card.route) cardWithPath.path = getGeneratedPath(card.route)
```

Comment:
Ah yeah, I'm in the camp that if statements with one instruction that fit on one line don't need curly braces. This is handy if you want to `return` out of a function for example. You only use 1 line instead of 3.

In the article you link the examples all show the actions are all on a new line below the if-statement. That's something I would never do or allow.

We can enforce this with this eslint rule: https://eslint.org/docs/latest/rules/nonblock-statement-body-position

Reply:
Interesting. You could of course use curly braces on the same line but if the eslinter can catch mistakes like these that mitigates a lot of the problems I have with it. I'll take a look. For now it should be fine (can you enable this rule if you haven't already?)